pkg/cli/mirror/mirror.go

```diff
@@ -605,7 +606,7 @@ func (o *MirrorOptions) mirrorToMirrorWrapper(ctx context.Context, cfg v1alpha2.
 	if err := bundle.MakeWorkspaceDirs(o.Dir); err != nil {
 		return err
 	}
-	meta, mapping, err := o.Create(ctx, cfg)
+	meta, mapping, allCatalogs, err := o.Create(ctx, cfg)
 	if err != nil {
```

Comment:
this feature probably needs oc-mirror to grow a new option to mirror the manifestlist and keep the legacy behavior as such.

Reply:
+1